tests/raftstore/util.rs

```diff
@@ -89,6 +89,7 @@ pub fn new_store_cfg() -> Config {
         max_leader_missing_duration: Duration::from_secs(3),
         report_region_flow_interval: 100, // 100ms
         raft_store_max_leader_lease: TimeDuration::milliseconds(25 * 10),
+        use_sst_file_snapshot: true,
```

Comment:
@siddontang The unit test part is fine. There are unit tests for both v1 and v2, tests for snap manager using v1 and v2, and tests for the compatibility of v1 -> v2.